app/models/user.rb

```diff
@@ -39,6 +39,8 @@ def self.from_omniauth(auth)
   end
 
   def tld_check
+    return true if ENV['GITHUB_CLIENT_ID'].present?
```

Comment:
let's revisit, as this doesn't allow `ensure_organization` to run, later in this method